x-pack/plugin/sql/qa/jdbc/build.gradle

```diff
@@ -0,0 +1,61 @@
+description = 'Integration tests for SQL JDBC driver'
+apply plugin: 'elasticsearch.build'
+
+// Avoid circular dependency
```

Comment:
@mark-vieira I was struggling to undertstand why we end up with circular dependency without this. Any idea?
Do you have a better suggestion? (I tried renaming the module to `jdbc-qa` instead of just `jdbc` which also works).

Reply:
I'm not completely certain (this is before my time), but I suspect Gradle get's confused if it finds multiple subprojects with identical GAV coordinates, so we make them unique to avoid any dependency resolution conflicts. Again, since this is a QA project that doesn't get published, this value is competely arbitrary.